src/screens/discover/DiscoverScreen.tsx

```diff
@@ -83,6 +83,7 @@ export default function DiscoverScreen() {
             setScrollY(e.nativeEvent.contentOffset.y);
           }}
           as={ScrollView}
+          scrollEventThrottle={32}
```

Comment:
This is going to set the `scrollY` state every 30ms — I believe without this prop it's only set once at the start of the scroll event.

I don't even think we need the `scrollY` state.

Its only purpose as far as I can see is to avoid calling ` ref.current?.scrollTo` when the list is already scrolled to the top. But it'd be more efficient to just call `ref.current?.scrollTo` regardless of whether the list is already scrolled to the top.

Reply:
reverted